docs/api.rst

```diff
@@ -6912,6 +6912,99 @@ Getting Compound balances
+   :reqjson int from_timestamp: Timestamp from which to query compound historical data. If not given 0 is implied.
+   :reqjson int to_timestamp: Timestamp until which to query compound historical data. If not given current timestamp is implied.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "interest_profit": {
+                  "0xA0B6B7fEa3a3ce3b9e6512c0c5A157a385e81056": {
+                      "eip155:1/erc20:0xc00e94Cb662C3520282E6f5717214004A7f26888": {
+                              "amount": "3.5",
+                              "usd_value": "892.5",
+                          },
+                       "eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F": {
+                              "amount": "250",
+                              "usd_value": "261.1",
+                      }
+                  },
+                  "0x1D7D7Eb7035B42F39f200AA3af8a65BC3475A237": {
+                      "eip155:1/erc20:0xE41d2489571d322189246DaFA5ebDe1F4699F498": {
+                          "amount": "0.55",
+                          "usd_value": "86.1"
+                      }
+                  }
+               },
+               "debt_loss": {
+                  "0x1D7D7Eb7035B42F39f200AA3af8a65BC3475A237": {
+                       "ETH": {
+                              "amount": "0.1",
+                              "usd_value": "30.5",
+                      }
+                  }
+               },
+               "liquidation_profit": {
+                  "0x1D7D7Eb7035B42F39f200AA3af8a65BC3475A237": {
+                       "ETH": {
+                              "amount": "0.00005",
+                              "usd_value": "0.023",
+                      }
+                  }
+               },
+               "rewards": {
+                  "0xA0B6B7fEa3a3ce3b9e6512c0c5A157a385e81056": {
+                      "eip155:1/erc20:0xc00e94Cb662C3520282E6f5717214004A7f26888": {
+                              "amount": "3.5",
+                              "usd_value": "892.5",
+                          },
+                  }
+               }
+          },
+          "message": ""
+      }
+
+   :resjson object interest_profit: A mapping of addresses to mappings of totals assets earned from lending over a given period
+   :resjson object debt_loss: A mapping of addresses to mappings of totals assets lost to repayment fees and liquidation over a given period.
+   :resjson object liquidation_profit: A mapping of addresses to mappings of totals assets gained thanks to liquidation repayments over a given period.
+   :resjson object rewards: A mapping of addresses to mappings of totals assets (only COMP atm) gained as a reward for using Compound over a given period.
+
+   :statuscode 200: Compound statistics succesfully queried.
+   :statuscode 409: User is not logged in. Or compound module is not activated.
```

Comment:
you wrote it "should not happen" but code exists that can return statuscode 400 so it should be in the docs